report_qweb_signer/models/ir_actions_report.py

```diff
@@ -39,9 +39,14 @@ def _certificate_get(self, res_ids):
         """Obtain the proper certificate for the report and the conditions."""
         if self.report_type != "qweb-pdf":
             return False
+        obj = self.env[self.model].browse(res_ids[0])
+        if "company_id" in obj:
+            company_id = obj.company_id.id
+        else:
+            company_id = self.env.user.company_id.id
```

Comment:
`self.env.company`?

Reply:
Yes, it is changed later on another commit (I just kept the original commit history from 14.0)